awx/main/models/inventory.py

```diff
@@ -2616,6 +2616,9 @@ def build_script_private_data(self, inventory_update, private_data_dir):
         group_patterns = '[]'
         group_prefix = 'foreman_'
         want_hostcollections = 'False'
+        want_ansible_ssh_host = 'False'
```

Comment:
https://github.com/ansible/awx/blob/30688ee28ba7772c14f1b5569e018e81cbbb5e4c/awx/plugins/inventory/foreman.ini.example#L188-L192

So host names are DNS, and if this is set true, then it'll use IP in the ansible_host var?

That sounds like something that really should be in the inventory plugin. I can not find anything like it right now.

Reply:
> That sounds like something that really should be in the inventory plugin. I can not find anything like it right now.

@jainnikhil30 - you mentioned that you were working on a PR for the plugin as well. Can you post a link here when you have something up so we can compare the changes in the plugin/script?

Reply:
@AlanCoding @jladdjr yup here is the PR https://github.com/ansible/ansible/pull/66407

Reply:
> So host names are DNS, and if this is set true, then it'll use IP in the ansible_host var?

@AlanCoding - my understanding of how this works is that enabling `want_ansible_ssh_host` will add a hostvar called `ansible_ssh_host` that is set to be equal to the foreman ip fact. If the foreman ip fact is not defined, then I don't think this field gets defined.

Reply:
![image](https://user-images.githubusercontent.com/4440360/72303519-6638ae00-3622-11ea-82b6-3e64d8981ec6.png)

^ this is what I get when I point `ansible-inventory` at a `.foreman.yml` config file with `want_ansible_ssh_host: True`